models/WebsiteSetting.php

```diff
@@ -311,4 +310,19 @@ public function clearDependentCache()
     {
         \Pimcore\Cache::clearTag('website_config');
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function delete()
+    {
+        $nameCacheKey = $this->getName() . '~~~' . $this->getSiteId() . '~~~' . $this->getLanguage();
```

Comment:
I think it would be good to have this in a `protected static function`, then we can share it with `getByName`.

Reply:
I'd leave it as it is for now, as it also has different parameters (just the `id`). Thanks!